build.gradle

```diff
@@ -384,8 +384,8 @@ dependencies {
 
   implementation group: 'com.google.guava', name: 'guava', version: '31.1-jre'
   implementation group: 'javax.el', name: 'javax.el-api', version: '3.0.0'
-  implementation group: 'com.launchdarkly', name: 'launchdarkly-java-server-sdk', version: versions.launchDarklySdk
-  implementation group:"org.yaml", name: "snakeyaml", version:"1.31"
+  implementation group: 'com.launchdarkly', name: 'launchdarkly-java-server-sdk', version: '5.10.2'
```

Comment:
Can we keep the reference to `versions.launchDarklySdk` and upgrade `launchDarklySdk` to `5.10.2`?